### ISSUE TITLE
Accessibility: wp.a11y.speak Announcing Old Actions

### ISSUE THREAD

**Issue by davidakennedy** [voldemortensen/menu-customizer#77]
When I add a menu, delete a menu or add a menu item, I hear that action via `wp.a11y.speak` plus the previous one I did. Not sure if this is a bug or not, but wanted to note it.

Screencast: https://cloudup.com/c6WZ9nXKGos

To reproduce:

1. Create Menu using Safari and Voiceover. (You'll hear: Menu created)
2. Next, add an item. (You'll hear: Menu created, Menu item added)
3. Save the menu.
4. Delete the menu. (You'll hear: Menu item added, Menu deleted)

Essentially, it also announces the thing you did previously.

**Comment by celloexpressions** [comment]
Sounds like a likely core bug with `wp.a11y.speak`, potentially that only affects the Customizer? The plugin definitely wouldn't be double-calling it based on where the code is.

**Comment by davidakennedy** [comment]
I dropped this into the accessibility channel on Slack, and I'll ping some other folks about it. We'll see if we can't test it some more to get to the bottom of things. Thanks!

**Comment by afercia** [comment]
We're currently using 'aria-relevant': 'all' that is:
all:	Equivalent to the combination of all values, "additions removals text".
may need to be changed in "additions text" by the way never noticed this testing with JAWS or NVDA. Sorry, I can't test with VoiceOver. Would need some serious test. Looks like VoiceOver is the only screen reader able to get the "removals"?

**Comment by afercia** [comment]
Maybe related: due to the Customizer iframe, there are 2 wp-a11y scripts and 2 live region with the same id. See screenshot. Is there any way to avoid this?

![doubled Speak]
(https://cldup.com/8rcVHybCCv.png)

**Comment by celloexpressions** [comment]
Could we move this to a core ticket for easier tracking? Looks like it's definitely an issue with the way speak is working in core in the Customizer. Theoretically the duplicate ids shouldn't be an issue since one's inside an iframe, but that could very well be it. Or the `aria-relevant` thing mentioned above sounds like a likely candidate.

**Comment by afercia** [comment]
Yup, was also thinking to ship a WP Speak version 2, would be a nice opportunity for a new testing round and for fixing bugs if any. I've made a quick and dirty plugin to log in the console the Speak messages, meant only for debugging and development. If you think it could be useful for you just ping me on Slack :) See below:
![Speak debugging]
(https://cldup.com/7eIjOJJWGE.png)

**Comment by celloexpressions** [comment]
Going to close this for now since it's a core issue. Let's try to get it fixed there before beta, though.
